components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -246,6 +247,61 @@ sealed class TrackingProtectionAction : BrowserAction() {
     data class ClearTrackersAction(val tabId: String) : TrackingProtectionAction()
 }
 
+/**
+ * [BrowserAction] implementations related to updating BrowserState.extensions] and TabState.extensions.
+ */
+sealed class WebExtensionAction : BrowserAction() {
+
+    /**
+     * [BrowserAction] implementations related to updating the
+     * [mozilla.components.browser.state.state.BrowserState.extensions]
+     */
+    sealed class BrowserState : WebExtensionAction() {
+        /**
+         * Installs a given [extension] to the default
+         * [mozilla.components.browser.state.state.BrowserState.extensions].
+         */
+        data class InstallWebExtension(val extension: WebExtensionState) :
```

Comment:
Now that we our delegate we can also dispatch this action.

Reply:
Dispatched :)